lib/todo_web/templates/user_reset_password/edit.html.heex

```diff
@@ -1,26 +1,24 @@
-<h1>Reset password</h1>
-
 <.form let={f} for={@changeset} action={Routes.user_reset_password_path(@conn, :update, @token)}>
```

Comment:
On the login and registration pages, I added `class='flex flex-col'` to the `.form` element. From what I can tell, this prevents the `m-2` classes from overlapping on consecutive form elements. Weird, I know. Would you add that to this form and the "Forgot your password?" form just so we are consistent?

Reply:
I added those classes.